django_fastdev/apps.py

```diff
@@ -243,11 +251,12 @@ def is_from_project(cls):
     if not module or not hasattr(module, "__file__"):
         return False
 
-    venv_dir = os.environ.get("VIRTUAL_ENV", "")
+    venv_dir = get_venv_path()
+    project_dir = get_path_for_django_project()
     module_path = os.path.abspath(module.__file__)
-    return module_path.startswith(
-        str(settings.BASE_DIR)
-    ) and not module_path.startswith(venv_dir)
+    return module_path.startswith(str(project_dir)) and not (
+        bool(venv_dir) and module_path.startswith(venv_dir)
```

Comment:
This isn't very readable imo. Shouldn't this be some thing like:

```py
if venv_dir is None or module_path.startswith(venv_dir):
    return False

return module_path.startswith(project_dir)
```
?

Reply:
ok I will reformat it, hold on